scripts/hf_eval.py

```diff
@@ -57,6 +59,16 @@ def run_evaluation(repo_id, tasks, limit, device, precision, quantization, compi
         quantize_(model.to(device=device), int4_weight_only())
     elif quantization == "autoquant":
         model = autoquant(model.to(device=device))
+    elif quantization == "fp8-static":
+        granularity = ScalingGranularity.TENSOR_WISE if granularity == "tensor" else ScalingGranularity.AXIS_WISE
+        model = model.to(device)
+        if granularity == ScalingGranularity.AXIS_WISE:
+            raise NotImplementedError("fp8-static with axis-wise granularity is not supported yet, since we need a good way to get the scales")
+        quantize_to_float8(model, QuantConfig(ActivationCasting.STATIC, torch.tensor([1.0], device=device), granularity,))
```

Comment:
is there a plan to refactor these with an api more closely aligned with AffineQuantizedTensor